### PR TITLE
adding authors, contributing, and license

### PR DESCRIPTION
**Pull request by kevmoo**
Preparing for the move to the google org

### CODE REVIEW

**Comment by Fox32** [comment]
@kevmoo Is it intentionally that there are now both AUTHORS and AUTHORS.txt? Maybe copy the old names and delete the old one?

**Comment by kevmoo** [comment]
Good call!

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Below is a list of people and organizations that have contributed
+# to the project. Names should be added to the list like so:
+#
+#   Name/Organization <email address>
+
+Google Inc.
+
+Brandon Jones <bajones@google.com>
```
Comment (johnmccutchan) [inline AUTHORS]: Remove @google.com addresses.

**Comment by kevmoo** [comment]
PTAL

**Pull request merged**